pyzscaler/zia/traffic.py

```diff
@@ -1,21 +1,30 @@
-from restfly.endpoint import APIEndpoint
-from pyzscaler.utils import snake_to_camel
 from box import BoxList
+from restfly.endpoint import APIEndpoint
+
+from pyzscaler.utils import Iterator, snake_to_camel
 
 
 class TrafficForwardingAPI(APIEndpoint):
-    def list_gre_tunnels(self):
+    def list_gre_tunnels(self, **kwargs):
         """
         Returns the list of all configured GRE tunnels.
 
+        Keyword Args:
+            **max_items (int, optional):
+                The maximum number of items to return before stopping iteration.
+            **max_pages (int, optional):
+                The maximum number of pages to request before throwing stopping iteration.
```

Comment:
Quality/Grammar: The maximum number of pages to request before stopping iteration.